docs/apps/nodes/add-validator.md

```diff
@@ -496,7 +496,7 @@ curl -s -X POST --data '{
 - **endTime**: The time that the validation period ends. It is the same format as `startTime`. In this example we used _2 days_ in the future.
   For Camino mainnet, you need minimum **2 weeks** and maximum **365 days** for the `endTime`.
 
-- **stakeAmount**: This is the _**fixed**_ amount of tokens to stake. For Camino testnet `columbus` it is `2000000000000` nCAMs (2000 CAMs). For mainnet it is `100000000000000` nCAMs (100k CAMs).
+- **stakeAmount**: This is the _**fixed**_ amount of tokens to stake. For Camino mainnet `camino` and testnet `columbus` it is `100000000000000` nCAMs (100k CAMs). (For devnet `kopernikus` it is 2k CAMs)
```

Comment:
should we give information about devnet in the public docs? I don't know but maybe ask Alex or Marcus.

Reply:
When you run local network for testing/developing etc, having the info is handy. Also I don't see what harm could it do. @Noctunus @peak3d any comments?

Reply:
I think it's OK to leave it for the devs